src/test/features/response/routes/check-and-send.ts

```diff
@@ -6,6 +6,7 @@ import { attachDefaultHooks } from '../../../routes/hooks'
 import '../../../routes/expectations'
 import { checkAuthorizationGuards } from './checks/authorization-check'
 import { checkAlreadySubmittedGuard } from './checks/already-submitted-check'
+import { checkCountyCourtJudgmentRequestedGuardGuard } from './checks/ccj-requested-check'
```

Comment:
It should be `checkCountyCourtJudgmentRequestedGuard` not `checkCountyCourtJudgmentRequestedGuardGuard`. Remove duplicate `Guard` from the name.

Reply:
well spotted